db/migrate/20190303143409_drop_product_distributions.rb

```diff
@@ -0,0 +1,37 @@
+class DropProductDistributions < ActiveRecord::Migration
+  def up
+    drop_table :product_distributions
```

Comment:
Did you run this against production data? Is there any dependent data destroyed?

Reply:
I tried it now, it just works.

Reply:
I tried with AUS data.